playbooks/roles/githubsshkeys/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+- name: install httplib2
+  sudo: true
+  pip: name=httplib2
+
+- name: get SSH keys from GitHub
+  uri: >
+    url=https://github.com/{{ item }}.keys
```

Comment:
Add quotes around the string since it contains curly braces (`url="https://github.com/{{ item }}.keys"`)